UnityProject/Assets/Scripts/Items/Implants/Organs/Bones.cs

```diff
@@ -5,8 +5,12 @@ namespace Items.Implants.Organs
 {
 	public class Bones : BodyPartFunctionality
 	{
-		[SerializeField] private float BloodGeneratedByOneNutriment = 1;
+		[SerializeField] public float BloodGeneratedByOneNutriment = 30;
 		[SerializeField] private BloodType GeneratesThis;
+
+		public float GenerationOvershoot = 1;
```

Comment:
Bod, we talked about public fields before. You know it is part of our code guidelines. Why do you insist on not respecting them?

Reply:
what part of it as it?